src/features/post/components/Category.tsx

```diff
@@ -0,0 +1,13 @@
+import {Badge} from "@/components/shadcn-ui/badge";
+import {FC} from "react";
+
+interface CategoryProps extends React.HTMLAttributes<HTMLElement> {
+  "data-testid": string;
```

Comment:
rename the component to 

CategoryBadge

and its API should look the same as shadcn-ui badge, except children must be a string